repos/iiif-storage/src/Model/BuiltCanvas.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace IIIFStorage\Model;
+
+
+class BuiltCanvas
```

Comment:
Would be nice to call this `Canvas`, since it's built by `CanvasBuilder`, but that leaves us with a naming conflict across another namespace.

Reply:
Once we drop the IIIF-PHP library, we could do that. Its strange because there's also `CanvasRepresentation` which wraps IIIF-PHP, JSON and the Omeka item which is passed to the templates!